{"name": "test_successful_login", "status": "broken", "statusDetails": {"message": "ImportError: cannot import name 'ChromeType' from 'webdriver_manager.core.utils' (C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\webdriver_manager\\core\\utils.py)", "trace": "@allure.tag(\"web\")\n    @allure.feature(f'Sign in')\n    @allure.story(\"Successful login and expected redirection of URL\")\n    def test_successful_login():\n        with allure.step('Select login and enter an valid username and password'):\n>           app.user_log_in.open() \\\n                .fill_username(user.username) \\\n                .fill_password(user.password) \\\n                .login_button()\n\ntest_login_page.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\sauce_demo_UI_autotests\\model\\pages\\login_page.py:6: in open\n    browser.open('/')\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\entity.py:1154: in open\n    self.config._executor.get_url(relative_or_absolute_url)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py:303: in get_url\n    self.config._driver_get_url_strategy(self.config)(url)\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py:218: in get\n    driver = config.driver\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py:345: in __get__\n    driver = config.build_driver_strategy(config)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nconfig = <selene.core.configuration.Config object at 0x0000027845D63C10>\n\n    def _build_local_driver_by_name_or_remote_by_url_and_options(\n        config: Config,\n    ) -> WebDriver:\n        from selenium.webdriver import (\n            ChromeOptions,\n            EdgeOptions,\n            Chrome,\n            Firefox,\n            Edge,\n        )\n    \n        from selenium.webdriver.chrome.service import Service as ChromeService\n        from selenium.webdriver.firefox.service import Service as FirefoxService\n        from selenium.webdriver.edge.service import Service as EdgeService  # type: ignore\n    \n        from webdriver_manager.chrome import ChromeDriverManager  # type: ignore\n        from webdriver_manager.firefox import GeckoDriverManager  # type: ignore\n        from webdriver_manager.microsoft import EdgeChromiumDriverManager  # type: ignore\n    \n>       from webdriver_manager.core.utils import ChromeType  # type: ignore\nE       ImportError: cannot import name 'ChromeType' from 'webdriver_manager.core.utils' (C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\webdriver_manager\\core\\utils.py)\n\n..\\..\\..\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py:69: ImportError"}, "steps": [{"name": "Select login and enter an valid username and password", "status": "broken", "statusDetails": {"message": "ImportError: cannot import name 'ChromeType' from 'webdriver_manager.core.utils' (C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\webdriver_manager\\core\\utils.py)\n", "trace": "  File \"C:\\Users\\skmee\\PycharmProjects\\sauce_demo_UI_autotests\\tests\\test_login_page.py\", line 25, in test_successful_login\n    app.user_log_in.open() \\\n    ^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\skmee\\PycharmProjects\\sauce_demo_UI_autotests\\sauce_demo_UI_autotests\\model\\pages\\login_page.py\", line 6, in open\n    browser.open('/')\n  File \"C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\entity.py\", line 1154, in open\n    self.config._executor.get_url(relative_or_absolute_url)\n  File \"C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py\", line 303, in get_url\n    self.config._driver_get_url_strategy(self.config)(url)\n  File \"C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py\", line 218, in get\n    driver = config.driver\n             ^^^^^^^^^^^^^\n  File \"C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py\", line 345, in __get__\n    driver = config.build_driver_strategy(config)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\skmee\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selene\\core\\configuration.py\", line 69, in _build_local_driver_by_name_or_remote_by_url_and_options\n    from webdriver_manager.core.utils import ChromeType  # type: ignore\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1690549118466, "stop": 1690549118856}], "start": 1690549118466, "stop": 1690549118867, "uuid": "78622ced-ee25-4be0-a830-85a9416074b6", "historyId": "5d6b3d5d62b0660f56728b293fa019d6", "testCaseId": "0946732b9b782b2ff605b607bfac2d22", "fullName": "tests.test_login_page#test_successful_login", "labels": [{"name": "feature", "value": "Sign in"}, {"name": "story", "value": "Successful login and expected redirection of URL"}, {"name": "tag", "value": "web"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_page"}, {"name": "host", "value": "LAPTOP-OFQSKCIR"}, {"name": "thread", "value": "12452-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_page"}]}